app/controllers/scim_rails/scim_users_controller.rb

```diff
@@ -82,13 +82,14 @@ def patch_update
       user = @company.public_send(ScimRails.config.scim_users_scope).find(params[:id])
 
       params["Operations"].each do |operation|
-        raise ScimRails::ExceptionHandler::UnsupportedPatchRequest if operation["op"] != "replace"
+        raise ScimRails::ExceptionHandler::UnsupportedPatchRequest unless ["Replace", "replace"].include?(operation["op"])
 
-        changed_attributes = permitted_params(operation["value"])
+        path_params = (operation.key?("path")) ? process_path(operation) : nil
+        changed_attributes = permitted_params(path_params || operation["value"])
 
         user.update!(changed_attributes.compact)
 
-        active_param = operation.dig("value", "active")
+        active_param = (operation.key?("path")) ? path_params&.dig(:active) : operation.dig("value", "active")
```

Comment:
I'd also extract this to its own method. Ternaries are nice, but only if they make it easier to read the code. Once you're 5+ lines into a method, ternaries wind up being harder to read.